Source/WebCore/history/CachedPage.cpp

```diff
@@ -77,7 +77,7 @@ CachedPage::~CachedPage()
         m_cachedMainFrame->destroy();
 }
 
-static void firePageShowEvent(Page& page)
+static void fireEvents(Page& page)
```

Comment:
This doesn't say what kind of events are getting fired.
Why not rename it to `firePageShowAndResumeEvent`.